docs/admin/ServerSideKeygen.adoc

```diff
@@ -84,6 +84,20 @@ output.o1.class_id=pkcs12OutputImpl
 
 === Policyset
 
+Password for the generated PKCS12 can be enforced with the follwing policy:
+
+[lietaral]
+policyset.userCertSet.11.constraint.class_id=p12ExportPasswordConstraintImpl
+policyset.userCertSet.11.constraint.name=PKCS12 Password Constraint
+policyset.userCertSet.11.constraint.params.password.minSize=20
+policyset.userCertSet.11.constraint.params.password.minCharCategory=2,2,2,2
```

Comment:
Didn't we get rid of the minCharCategory and use individual params instead?

Reply:
Yes, I took from the wrong policy. I am fixing. Thanks